### DISCUSSION TITLE
Jyers Manual 5x5 on Ender 3 Pro

### DISCUSSION THREAD

**Discussion by KanuCanoe** [Jyers/Marlin#1284] [Q&A] [answered]
I am wanting to use the manual mesh bed leveling for my Ender 3 Pro and I keep getting a solid blue screen. I am going to assume it has something to do with the difference in screens between the v2 and the pro. I have the 4.2.2 board so that shouldn't be the issue. What do I need to change in the firmware for it to work with the stock pro screen? I will figure out how to change and compile the firmware after I know what needs changing.

Thank you for any help.

**Comment by Cal-LF** [comment]
Jyers FW is a menu for the E3V2 screen. It does not work with the Pro screen or a touch screen. It does work if you change to the E3V2 screen and your V422 mainboard.

**Reply by KanuCanoe** [reply]
I have the firmware on my v2, it seems to unlock better leveling and manual mesh leveling which are not open in the standard Pro firmware. So this firmware does more than just make new menus appear on a particular screen. I'm just looking for what I would need to change in the firmware to use the non-color screen of the Pro.

**Comment by Cal-LF** [comment, marked as answer]
@KanuCanoe As I said, it will not work on the Pro screen. Put a V2 screen on the Pro to get it working.
